flink-java/src/main/java/org/apache/flink/api/java/ExecutionEnvironment.java

```diff
@@ -118,6 +119,9 @@
 
 	private final ExecutionConfig config = new ExecutionConfig();
 
+	private final BlockingPersistentResultPartitionMeta blockingPersistentResultPartitionMeta =
```

Comment:
This should probably not be in a separate field, but in the `lastJobExecutionResult`.

Reply:
I think this should be a separate field, since it keeps track of all descriptors from all previous jobs rather than the latest one.